app/uk/gov/hmrc/tradergoodsprofiles/models/responses/CreateRecordResponse.scala

```diff
@@ -0,0 +1,133 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tradergoodsprofiles.models.response
+
+import play.api.libs.json._
+import uk.gov.hmrc.tradergoodsprofiles.models.Assessment
+
+import java.time.Instant
+
+case class CreateRecordResponse(
+  recordId: String,
+  eori: String,
+  actorId: String,
+  traderRef: String,
+  comcode: String,
+  accreditationStatus: String,
+  goodsDescription: String,
+  countryOfOrigin: String,
+  category: Int,
+  assessments: Seq[Assessment],
+  supplementaryUnit: Int,
+  measurementUnit: String,
+  comcodeEffectiveFromDate: Instant,
+  comcodeEffectiveToDate: Instant,
+  version: Int,
+  active: Boolean,
+  toReview: Boolean,
+  reviewReason: Option[String],
+  declarable: String,
+  ukimsNumber: String,
+  nirmsNumber: String,
+  niphlNumber: String,
+  createdDateTime: Instant,
+  updatedDateTime: Instant
+)
+
+object CreateRecordResponse {
+  implicit lazy val format: OFormat[CreateRecordResponse] = new OFormat[CreateRecordResponse] {
+    override def writes(o: CreateRecordResponse): JsObject =
+      Json.obj(
+        "recordId"                 -> o.recordId,
+        "eori"                     -> o.eori,
+        "actorId"                  -> o.actorId,
+        "traderRef"                -> o.traderRef,
+        "comcode"                  -> o.comcode,
+        "accreditationStatus"      -> o.accreditationStatus,
+        "goodsDescription"         -> o.goodsDescription,
+        "countryOfOrigin"          -> o.countryOfOrigin,
+        "category"                 -> o.category,
+        "assessments"              -> o.assessments,
+        "supplementaryUnit"        -> o.supplementaryUnit,
+        "measurementUnit"          -> o.measurementUnit,
+        "comcodeEffectiveFromDate" -> o.comcodeEffectiveFromDate,
+        "comcodeEffectiveToDate"   -> o.comcodeEffectiveToDate,
+        "version"                  -> o.version,
+        "active"                   -> o.active,
+        "toReview"                 -> o.toReview,
+        "reviewReason"             -> o.reviewReason,
+        "declarable"               -> o.declarable,
+        "ukimsNumber"              -> o.ukimsNumber,
+        "nirmsNumber"              -> o.nirmsNumber,
+        "niphlNumber"              -> o.niphlNumber,
+        "createdDateTime"          -> o.createdDateTime,
+        "updatedDateTime"          -> o.updatedDateTime
+      )
+
+    override def reads(json: JsValue): JsResult[CreateRecordResponse] = for {
+      recordId                 <- (json \ "recordId").validate[String]
```

Comment:
I've just seen this. We don't need to validate response body, so this validate is not needed.
This file can be made to mimic this https://github.com/hmrc/trader-goods-profiles-router/blob/main/app/uk/gov/hmrc/tradergoodsprofilesrouter/models/response/CreateOrUpdateRecordResponse.scala